src/plugins/buflist/buflist-config.c

```diff
@@ -645,7 +648,7 @@ buflist_config_init ()
            "[${number}]}\""),
         NULL, 0, 0,
         "${format_number}${indent}${format_nick_prefix}${color_hotlist}"
-        "${format_name}",
+        "${format_name}${format_tls_version}",
```

Comment:
I wonder if this should be displayed by default. I think most people doesn't want this by default, but it's hard to know what people prefer.
Personally, I think it's better to not have this by default, to keep a simple format by default.

Reply:
I agree, this should not be a default value.

Reply:
Removed. Weechat is known for its good defaults so I think you have better taste here.